base/bionic/Dockerfile

```diff
@@ -24,7 +26,8 @@ RUN apt-get update -qq && \
     sudo \
     tzdata \
     wget \
-    zlib1g-dev
+    zlib1g-dev && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
similarly, when you `wget` files, delete them after installing their contents. If you expand a tarball/zipfile in order to build something, delete the directory after building & installing.

Reply:
I looked at each wget instruction but didn't see any leftover cruft -- are we missing something?. For TinyTeX, it dumps a script to stdout; for pandoc, `gzip -d` automatically deletes the 2 tarballs; and for the R binary, it `rm`s the tarball afterward.

Reply:
TIL about `gzip`.